pkgs/development/python-modules/harlequin-mysql/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  poetry-core,
+  python3Packages,
+}:
+
+buildPythonPackage rec {
+  pname = "harlequin-mysql";
+  version = "0.3.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "harlequin_mysql";
+    inherit version;
+    hash = "sha256-Ru9CxbZYVo9TQO5TwkHLEzPz4EkUgHwfg3Qeg1F4eLM=";
+  };
+
+  build-system = [
+    poetry-core
+  ];
+
+  dependencies = with python3Packages; [
+    mysql-connector
+  ];
+
+  pythonRelaxDeps = [
+    "mysql-connector-python"
+  ];
+
+  # To prevent circular dependency
+  # as harlequin-mysql requires harlequin which requires harlequin-mysql
+  doCheck = false;
+  pythonRemoveDeps = [
+    "harlequin"
+  ];
+
```

Comment:
Could you enable `pythonImportsCheck` for a smoke test?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck